pages/popup.css

```diff
@@ -0,0 +1,14 @@
+body {
+	background-color: #FFF;
+	margin: 0px;
+	padding: 0px;
+	width: 130px;
+}
+
+* {
+	-webkit-user-select: none;
```

Comment:
Woa! Is this the absolutely required for PhotonUI or something? I always wondered how some sites annoyed me that way.